book/styles.md

```diff
@@ -839,75 +912,55 @@ def paint(self, display_list):
         display_list.append(DrawText(x, y, word, font, color))
 ```
 
-Phew! That was a lot of coordinated changes, so please test everything
-and make sure it works. You should now see links on this page appear
-in blue---and you'll also notice that the rest of the text has become
-slightly lighter.[^book-css]
-
-[^book-css]: Check out [the book's style sheet](book.css) to see the
-    details.
-
-Font Properties
-===============
-
-Since we're already mucking around with `InlineLayout`, let's also
-modify it to look up the font size, weight, and style information via
-CSS instead of using the `style`, `weight`, and `size` fields on
-`InlineLayout`:
+`DrawText` now needs a `color` argument, and needs to pass it to
+`create_text`'s `fill` parameter:
 
-::: {.todo}
-How do we resolve percentage values for nodes? And the `font-size`
-needs to be converted from pixels to points.
-:::
-[^72ppi]
+``` {.python}
+class DrawText:
+    def __init__(self, x1, y1, text, font, color):
+        # ...
+        self.color = color
 
-[^72ppi]: Normally you think of points as a physical length unit (one 72^nd^ of
-an inch) and pixels as a digital unit (dependent on the screen) but in CSS, the
-conversion is fixed at exactly 75% (or 96 pixels per inch) because that was once
-a common screen resolution. This might seem weird, but [OS internals][why-72]
-are equally bizarre, let alone the fact that a traditional typesetters' point is
-[one 72.27^th^ of an inch][why-7227].
+    def execute(self, scroll, canvas):
+        canvas.create_text(
+            # ...
+            fill=self.color,
+        )
+```
 
-[why-72]: https://tonsky.me/blog/font-size/
-[why-7227]: https://tex.stackexchange.com/questions/200934/why-does-a-tex-point-differ-from-a-desktop-publishing-point
+Phew! That was a lot of coordinated changes, so test everything and
+make sure it works. You should now see links on this page appear in
+blue---and you'll might also notice that the rest of the text has
+become slightly lighter.[^book-css]
 
-``` {.python indent=4}
-def text(self, node):
-    # ...
-    weight = node.style["font-weight"]
-    style = node.style["font-style"]
-    if style == "normal": style = "roman"
-    size = node.style["font-size"] * .75 # ???
-    font = tkinter.font.Font(size=style, weight=weight, slant=style)
-    # ...
-```
+[^book-css]: The book's main body text [is colored](book.css) `#333`,
+    or roughly 97% black.
```

Comment:
It is? 3/15 = 20%, no?

Reply:
Yeah, but then it's gamma corrected. I actually busted out the sRGB converter and checked. Perhaps I should say something like "97% black after gamma correction" and link "gamma correction" to the Wikipedia page?

Reply:
SGTM